src/redraw.mm

```diff
@@ -360,14 +303,14 @@ - (void) doAction:(RedrawCode::Enum)code withArgc:(int)argc argv:(const object *
             CGSize size = bitmapContextSizeInPoints(self, mCanvasContext);
             NSRect totalRect = {CGPointZero, size};
 
-            totalRect.origin.y += amt * mCharSize.height;
+            totalRect.origin.y = floor(totalRect.origin.y + amt * mCharSize.height);
 
             CGContextSaveGState(mCanvasContext);
             CGContextClipToRect(mCanvasContext, dest);
             drawBitmapContext(mCanvasContext, mCanvasContext, totalRect);
             CGContextRestoreGState(mCanvasContext);
 
-            /* Clear the newly-visible lines */
+            // Clear the newly-visible lines
```

Comment:
I'm not too fussed about what comment style you use when writing code; I just prefer for PRs not to have changes that don't relate to the issue at hand, such as changing the style of existing comments. I'll get on with that CONTRIBUTE file :)